src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3587,6 +3557,90 @@ namespace ReportSizingManager {
         }
     }
 
+    Real64 setOAFracForZoneEqSizing(Real64 const &desMassFlow, DataSizing::ZoneEqSizingData const &zoneEqSizing)
+    {
+        Real64 outAirFrac = 0.0;
+        if (desMassFlow <= 0.0) return outAirFrac;
+
+        if (zoneEqSizing.ATMixerVolFlow > 0.0) {
+            // set central DOAS AT mixer OA fraction
+            outAirFrac = min(DataEnvironment::StdRhoAir * zoneEqSizing.ATMixerVolFlow / desMassFlow, 1.0);
+        } else if (zoneEqSizing.OAVolFlow > 0.0) { // set zone equipment OA fraction
+            outAirFrac = min(DataEnvironment::StdRhoAir * zoneEqSizing.OAVolFlow / desMassFlow, 1.0);
+        }
+        return outAirFrac;
+    }
```

Comment:
@Myoldmopar I changed this one just to get familiar. It works as expected. I didn't change the other 4 below since I ran out of time. I would need 2 additional arguments for those below and then the function calls in RequestSizing would get messier.

Reply:
Obviously I circled back and fixed the remaining functions below.